src/Automation/ScanResultsAssembler.cs

```diff
@@ -37,7 +38,7 @@ public static ScanResults AssembleScanResultsFromElement(A11yElement element)
         /// <param name="errors">Where the <see cref="ScanResult"/> objects created from errors will be added</param>
         /// <param name="element">Root element from which errors will be assembled</param>
         /// <param name="parent">The parent of element</param>
-        internal static void AssembleErrorsFromElement(List<ScanResult> errors, A11yElement element, ElementInfo parent)
+        internal void AssembleErrorsFromElement(List<ScanResult> errors, A11yElement element, ElementInfo parent)
```

Comment:
I think you can actually leave the private static functions static because they don't rely on any member data.

Reply:
Restored non-public functions to be static